Date/Span.php

```diff
@@ -1125,7 +1125,7 @@ function getDefaultInputFormat()
      * @static
      * @see      Date_Span::getDefaultFormat(), Date_Span::setDefaultInputFormat()
      */
-    function setDefaultFormat($format)
+    static function setDefaultFormat($format)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.